client/src/AllVideos.css

```diff
@@ -0,0 +1,199 @@
+* {
```

Comment:
I'd put a reset like this in the top level stylesheet eg App.css, you wouldn't normally find resets in component stylesheets

Reply:
Oh I see, if I do have it at the top level, would I then need to import the App.css for instance to this component?

Reply:
No, as long as the top level stylesheet is loaded into the page somewhere, it doesn't need to be directly imported into this stylesheet. All the selectors will be considered together.